modules/lvgl/Kconfig

```diff
@@ -153,6 +153,31 @@ config LV_Z_INIT_PRIORITY
 	help
 	  Priority used for the automatic initialization of LVGL.
 
+config LV_Z_HANDLER_THREAD
+	bool "Drive lv_timer_handler in a separate thread"
```

Comment:
Nit: something like "Run LVGL event loop in a seperate thread" more resilient against API renaming.